catalog/src/main/kotlin/com/adevinta/spark/catalog/model/Components.kt

```diff
@@ -219,6 +221,19 @@ private val Progressbars = Component(
     configurator = ProgressbarConfigurator,
 )
 
+private val TextLinks = Component(
+    id = nextId(),
+    name = "TextLinks",
+    description = R.string.component_textlink_description,
+    illustration = R.drawable.icon_textlink,
+    tintIcon = false,
+    guidelinesUrl = "$ComponentGuidelinesUrl/p/75ed11-textlink/b/403107",
+    docsUrl = "$PackageSummaryUrl/com.adevinta.spark.components.text/index.html",
+    sourceUrl = "$SparkSourceUrl/kotlin/com/adevinta/components/tetx/TextLink.kt",
```

Comment:
This looks like a typo
```suggestion
    sourceUrl = "$SparkSourceUrl/kotlin/com/adevinta/components/text/TextLink.kt",
```